test/sphinxapi.py

```diff
@@ -225,7 +225,7 @@ def _Connect (self):
                 addr = ( self._host, self._port )
                 desc = '%s;%s' % addr
             sock = socket.socket ( af, socket.SOCK_STREAM )
-            sock.settimeout ( self._timeout )
+            sock.settimeout ( 200 )
```

Comment:
why is it hardcoded? Wouldn't be better to keep self._timeout if you add at line 119 an argument to the constructor?